libs/agno/agno/models/base.py

```diff
@@ -145,6 +145,9 @@ class Model(ABC):
     cache_ttl: Optional[int] = None
     cache_dir: Optional[str] = None
 
+    # Context compression manager
+    context_manager: Optional[Any] = None
```

Comment:
We cannot have state like this on the model class. It has to be passed through to the model